OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -250,13 +259,18 @@ Action InitDisplayPanel(Widget panel)
 				"Windowed" : ds.Mode == WindowMode.Fullscreen ? "Fullscreen (Legacy)" : "Fullscreen";
 
 			var modeChangesDesc = panel.Get("MODE_CHANGES_DESC");
-			modeChangesDesc.IsVisible = () => ds.Mode != WindowMode.Windowed && ds.Mode != OriginalGraphicsMode;
+			modeChangesDesc.IsVisible = () => ds.Mode != WindowMode.Windowed && (ds.Mode != OriginalGraphicsMode ||
+				ds.VideoDisplay != OriginalVideoDisplay);
 
 			var statusBarsDropDown = panel.Get<DropDownButtonWidget>("STATUS_BAR_DROPDOWN");
 			statusBarsDropDown.OnMouseDown = _ => ShowStatusBarsDropdown(statusBarsDropDown, gs);
 			statusBarsDropDown.GetText = () => gs.StatusBars == StatusBarsType.Standard ?
 				"Standard" : gs.StatusBars == StatusBarsType.DamageShow ? "Show On Damage" : "Always Show";
 
+			var displaySelectionDropDown = panel.Get<DropDownButtonWidget>("DISPLAY_SELECTION_DROPDOWN");
+			displaySelectionDropDown.OnMouseDown = _ => ShowDisplaySelectionDropdown(displaySelectionDropDown, ds);
+			displaySelectionDropDown.GetText = () => string.Format("{0} {1}", DisplayBaseName, ds.VideoDisplay);
```

Comment:
Style nits: OpenRA code style is to use `"{0} {1}".F(DisplayBaseName, ds.VideoDisplay)`, and this should also be wrapped in a `CachedTransform<int, string>` keyed on `ds.VideoDisplay` to avoid recreating this string 60 times per second.